Machine Learning/sklearn examples/K Mean Clustering/K-Mean Clustering .ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
<p>I am sorry, I think I missed these in the first review. Could you remove the bullet before </p><blockquote>Notebook is contributed by: aditya9110</blockquote><p>to maintain consistency. Also, once you do this I will make some post-merge edits (ideally nothing more would be required from your end)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>